mono/profiler/mono-profiler-logging.suppression

```diff
@@ -0,0 +1,149 @@
+Commons.Xml.Relaxng
```

Comment:
Ideally it would go inside mcs/ but I don't really see any directory there that would be applicable and it seems like overkill to add a whole directory for one auto generated file but I can do that on a separate branch so that it doesn't disrupt this PR